### DISCUSSION TITLE
Add python syntax markdown

### DISCUSSION THREAD

**Discussion by eeriemyxi** [stoatchat/discussions#255] [Feature Suggestions]
```python
print("basically add colors to markdowns like this")
```

**Comment by aeristhy** [comment]
iirc, a moderator in Revolt said that it's not a feature (I got a list of all the available syntax [here](https://github.com/aeristhy/Revolt-Syntax-Highlighting))

**Comment by cyrneko** [comment]
I'm pretty sure that if Markdown works that you could also do `html` with a little bit of work

**Comment by Rexogamer** [comment]
Apologies for the late reply, but Revite [has supported syntax highlighting for Python](https://github.com/revoltchat/revite/blob/f14ef2b78f39482fdc9ea04875ae6b1633759a8b/src/components/markdown/prism.ts#L32) since December
